### DISCUSSION TITLE
helm charts published as release assets

### DISCUSSION THREAD

**Discussion by farazkhawaja** [vmware-tanzu/velero#8268] [Community support Q&A] [answered]
Hi, can we have helm charts manifest files such as crd's yaml and deployment.yaml etc be posted in release assets tab.

**Comment by kaovilai** [comment, marked as answer]
While not in assets, you can grab latest release tag, and get yaml from there.

https://github.com/vmware-tanzu/helm-charts/tree/velero-7.2.1/charts/velero/crds

like https://raw.githubusercontent.com/vmware-tanzu/helm-charts/refs/tags/velero-7.2.1/charts/velero/crds/backupstoragelocations.yaml

**Reply by farazkhawaja** [reply]
thanks.
